doc/rippled-example.cfg

```diff
@@ -1076,6 +1076,13 @@ file_size_mult=2
 online_delete=2000
 advisory_delete=0
 
+# This is the persistent datastore for shards. Helpful information can be
+# found here: https://ripple.com/build/history-sharding
+#[shard_db]
+#type=NuDB
+#path=/var/lib/rippled/db/shards/nudb
+#max_size_gb=500
```

Comment:
How about enabling a 20-50GB shard_db in the config below?

Reply:
Updated the default to be enabled.